packages/siopv2-oid4vp-op-auth/src/machine/Siopv2Machine.ts

```diff
@@ -74,13 +88,17 @@ const Siopv2IsSiopOnlyGuard = (_ctx: Siopv2MachineContext, _event: Siopv2Machine
   return authorizationRequestData.presentationDefinitions === undefined
 }
 
-const Siopv2IsSiopWithOID4VPGuard = (_ctx: Siopv2MachineContext, _event: Siopv2MachineEventTypes): boolean => {
-  const { authorizationRequestData } = _ctx
+const Siopv2CanDoSelectCredentialsGuard = (_ctx: Siopv2MachineContext, _event: Siopv2MachineEventTypes): boolean => {
```

Comment:
The previous name was a better name, also be aware whenever you rename machine guards and services it impacts multiple other dependent libs/products.

Reply:
Reverted, but personally isSiopWithOID4VPGuard doesn't tell me anything.

Reply:
Yeah, means you should read these specs, because it should be clear. The whole point is that it is called because a RP supports certain protocols or not, instead of the name suggesting whether the wallet could do something or not